_analyzers/token-filters/classic.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: default
+title: classic
+parent: Token filters
+nav_order: 50
+---
+
+# Classic token filter
+
+The primary function of the classic token filter is to work alongside the classic tokenizer. It processes tokens by applying several common transformations. These transformations aid in text analysis and search. These include:
+ - Removal of possessive endings such as *'s*, for example: *John's* becomes *John*.
+ - Separating words on internal hyphens, making terms like *co-operate* become tokens *co* and *operate*.
```

Comment:
I donot think this statement is correct wrt to classic token filter. 

I believe words are already separated when input is streamed through classic tokenizer not when passed through classic token filter. 

Below example shows words separation done via tokenizer

```
PUT my-index-classic2
{
  "settings": {
    "analysis": {
      "analyzer": {
        "custom_classic": {
          "type": "custom",
          "tokenizer": "classic"
        }
      }
    }
  }
}

GET my-index-classic2/_analyze
{
  "analyzer": "custom_classic", 
  "text": "John's co-operate was excellent."
}
```
Response
```
{
  "tokens": [
    {
      "token": "John's",
      "start_offset": 0,
      "end_offset": 6,
      "type": "<APOSTROPHE>",
      "position": 0
    },
    {
      "token": "co",
      "start_offset": 7,
      "end_offset": 9,
      "type": "<ALPHANUM>",
      "position": 1
    },
    {
      "token": "operate",
      "start_offset": 10,
      "end_offset": 17,
      "type": "<ALPHANUM>",
      "position": 2
    },
    {
      "token": "was",
      "start_offset": 18,
      "end_offset": 21,
      "type": "<ALPHANUM>",
      "position": 3
    },
    {
      "token": "excellent",
      "start_offset": 22,
      "end_offset": 31,
      "type": "<ALPHANUM>",
      "position": 4
    }
  ]
}
```

Reply:
This is correct, I have removed this now